src/lib/api/loans/loan.js

```diff
@@ -242,7 +242,7 @@ class QueryBuilder {
   }
 
   sortByRequestStartDate() {
-    this.sortBy = `&sort=-request_start_date`;
+    this.sortBy = `&sort=request_start_date`;
```

Comment:
there is a couple of problems I can see here.
1. this function can be reused somewhere else. By changing it in here you are changing it everywhere else. I would test what is being affected.
2. the name of the method is misleading, as you can see someone used it for sorting by decreasing date
3. since the sorting has an impact for displaying a modal (or not) for the proceeding loan request I would create a custom method (or even better, I would handle the sorting in the component which introduces the dependency with sorting and the modal)
for ex:

```suggestion
sortByRequestStartDateDesc() {
    this.sortBy = `&sort=-request_start_date`;
}

sortByRequestStartDateAsc() {
    this.sortBy = `&sort=request_start_date`;
}

sortToDetectPrecedingRequest() {
    this.sortBy = `&sort=request_start_date`;
}
```

Reply:
I prefer to discuss this IRL.
For clarity I did check if it was used elsewhere (it wasn't) but I agree that naming was confusing. I would merge this PR https://github.com/inveniosoftware/react-invenio-app-ils/pull/507 before changing this, as they affect the same area.